src/sql/workbench/parts/tasks/browser/media/tasksPanel.css

```diff
@@ -40,7 +40,7 @@
 	background-position: left center;
 	background-repeat: no-repeat;
 	padding-right: 6px;
-	width: 22px;
+	min-width: 22px;
```

Comment:
Any ideas why min-width would fix it? Its a better approach than the padding, but I'm still not quick sure I understand what the original problem was.